README.md

```diff
@@ -142,6 +143,36 @@ The provided `dcos.yml` playbook can be used as-is for installing and upgrading
 * CentOS 7, RHEL 7
 * DC/OS 1.12, both open as well as enterprise version
 
+### Using Flatcar Linux
+
+[Flatcar Container Linux](https://www.flatcar-linux.org/) is a minimal linux distribution designed to run containers. Due to its immutable design some additional work is needed to run the DC/OS ansible installer.
+
+As flatcar does not have python preinstalled you need to install it some other way. We recommend [ansible-flatcar-python-bootstrap](https://github.com/swoehrl-mw/ansible-flatcar-python-bootstrap). To use it add the role to your project (e.g. by running `ansible-galaxy install git+https://github.com/swoehrl-mw/ansible-flatcar-python-bootstrap`) and add the following snippet to the `dcos.yml` playbook before the `Collect DC/OS versions` play:
```

Comment:
Both `ansible_python_interpreter ` and `pip_executable` can be set in the inventory.

```
[bootstraps]
bootstrap1-dcos112s.example.com   ansible_user=core ansible_python_interpreter=/home/core/bin/python pip_executable=/home/core/pypy/bin/pip
```

Regarding including the bootstrap in the main playbook: What about protecting the bootstrap `include_role` with a `when: needs_portable_python` and having the user set that variable on all flatcar hosts in the inventory?  That way it can be a normal part of the playbook without branches/different Dockerfiles. Automatic detection would be tricky as `ansible_distribution` and co are not yet available at that point. Maybe something with a raw shell command greping `/etc/os-release` might be possible (but not pretty).

Reply:
Yeah that might be a good idea.